cdap-docs/integrations/source/index.rst

```diff
@@ -23,7 +23,7 @@ Integrations
 - |hub-api|_ Hub APIs used to create entities in a Hub
 
 .. |hub-custom| replace:: **Custom Hosting:**
-.. hub-custom: hub.html#custom
+.. _hub-custom: hub.html#custom
```

Comment:
is this necessary?

Reply:
yes; the underscore is something I accidentally removed in the previous commit.